src/Microsoft.Framework.Caching.SqlServer/project.json

```diff
@@ -0,0 +1,29 @@
+{
+    "commands": {
+        "sqlservercache": "Microsoft.Framework.Caching.SqlServer"
+    },
+    "dependencies": {
+        "Microsoft.Framework.Caching.Abstractions": "1.0.0-*",
+        "Microsoft.Framework.Logging": "1.0.0-*",
+        "Microsoft.Framework.Logging.Abstractions": "1.0.0-*",
```

Comment:
Actually I need the non-abstractions logging as I need to instantiate `LoggerFactory` to which the console logger provider is added.